Common/Inc/config.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "can_driver.h"
+#define MAX_CAN_PAYLOAD_BTYES    8
+#define BUFF_SIZE 32
+
+#define WARN_OFFSET 0
+
+//MOTOR CONTROLLER
+#define MOTOR_CONTROLLER_1     0x10F8109A
+#define MOTOR_CONTROLLER_2     0x10F8108D
+//BMS
+#define BMS_TEMP             0x10
+#define BMS_WARN_OVERTEMP    0x06
+//SENSORS
+#define SENSOR_BOARD         0x10 // temp
+#define SENSOR_BOARD_WARN     SENSOR_BOARD - WARN_OFFSET
+
+
+// BEGIN MOTOR CONTROLLER BOARD DEFS
+// FRAME 1
+const DataSegment DRIVING_DIRECTION = {MOTOR_CONTROLLER_1, 1, 1};
+const DataSegment MOTOR_SPEED = {MOTOR_CONTROLLER_1, 2, 3};
+const DataSegment MOTOR_ERROR_CODE = {MOTOR_CONTROLLER_1, 4, 4}; 
+//FRAME 2
+const DataSegment BATTERY_VOLTAGE = {MOTOR_CONTROLLER_2, 1, 2};
+const DataSegment BATTERY_CURRENT = {MOTOR_CONTROLLER_2, 3, 4};
+const DataSegment MOTOR_TEMP = {MOTOR_CONTROLLER_2, 5, 6};
+const DataSegment MOTOR_CONTROLLER_TEMP = {MOTOR_CONTROLLER_2, 7, 8};
+//END MOTOR CONTROLLER BOARD DEFS
+
+// BEGIN BMS DEFS
+// BEGIN DEFS FOR THERMISTOR MESSAGE
+const DataSegment MUX1_TEMP = {BMS_TEMP, 1, 1};
+const DataSegment MUX2_TEMP = {BMS_TEMP, 2, 2};
+const DataSegment MUX3_TEMP = {BMS_TEMP, 3, 3};
+const DataSegment MUX4_TEMP = {BMS_TEMP, 4, 4};
+const DataSegment MUX5_TEMP = {BMS_TEMP, 5, 5};
+const DataSegment MUX6_TEMP = {BMS_TEMP, 6, 6};
+// END DEFS FOR THERMISTOR MESSAGE
+
+// BEGIN DEFS FOR ERROR STATES
+const DataSegment MUX1_TEMP_WARN = {BMS_WARN_OVERTEMP, 1, 1};
```

Comment:
Don't need separate warn segments since the WARN_OFFSET implementation deals with that

Reply:
I think we still do. Don't we need the other boards to recognize the ID?